contracts/interchain-token-service/src/contract.rs

```diff
@@ -238,16 +240,39 @@ impl InterchainTokenServiceInterface for InterchainTokenService {
     }
 
     fn deploy_remote_interchain_token(
-        _env: &Env,
-        _caller: Address,
-        _salt: BytesN<32>,
-        _minter: Option<Bytes>,
-        _destination_chain: String,
-        _gas_token: Token,
+        env: &Env,
+        caller: Address,
+        salt: BytesN<32>,
+        minter: Option<Address>,
```

Comment:
Let's remove the minter in this method. We'll just add a separate method in another PR that allows specifying the minter because this needs to be bytes since other chains have a different address format